### ISSUE TITLE
Color prop - Box shadow - paste/type rgb(a) values broken/unattainable

### ISSUE THREAD

**Issue by pingram3541** [pojome/elementor#1127]
Running current versions of Elementor (1.0.12) and Pro (1.0.7), tested on 2 client sites w/ different hosting environments and found that when attempting to manually type or paste into the box-shadow color property the input is interrupted and one cannot complete the entry and is forced to used the visual color spectrum to "best guess" at the color.

One way I found around this is to save the color to my global color palette and then use that to get the exact rgba value I was seeking. (quicker when working with multiple elements that may need this style treatment)

If I type rgba(0 it stops right there and won't accept any more characters.

Strange thing is this seems to only happen when editing the box-shadow color property.  Other color props seem to work as expected outside the occasional auto formatting if one isn't fast enough.

For example when one types rgba(0,0,0 in other color properties it will quickly change to #000000, unless you enter the characters at just the right speed and pause entry in just the right places.  This goes for entering in hex too.

I think some refinements should be made to the color property overall, maybe retard the auto formatting timing mechanism so we have enough time to enter a proper and complete value without being curtailed by the software so quickly.

**Comment by drawcard** [comment]
This is most likely caused by the Wordpress Iris script that powers the colour picker. I've raised a similar issue in the past: https://github.com/pojome/elementor/issues/1026

I'd suggest hitting up the Iris github page and bringing this to their attention.

**Comment by pingram3541** [comment]
Hold up, I think this was closed out too fast.

I realize this may have been ported over from Iris and likely also using one of the 3rd party rgba fix - [https://github.com/Automattic/Iris/issues/13](https://github.com/Automattic/Iris/issues/13), but what I reported is only specific to BOX-SHADOW properties.

How could that not be native to this specific control type settings when it works as expected with other Elementor controls?

Also, I saw mention of the short horizontal space cutting off rgba values in which one cannot easily see the "b" and "opacity" values that @drawcard mentioned in another thread, can't that be fixed with simple css?
